tests/cases/fourslash/mapCodeClassInvalidClassMember.ts

```diff
@@ -0,0 +1,13 @@
+///<reference path="fourslash.ts"/>
+
+// @Filename: /incomingChanges.ts
+//// if (false) {
+////     return "hello";
+//// }
+////
```

Comment:
This should not be a file but input to test since problem is that this will be part of the program and you are not looking for that. So it should be verify.baselineMapCode(fileName, focusLocations, changes) 

Then add some test cases where fileName is `index.js` instead of `index.ts`

Reply:
So you want me to write an inline string with code instead of doing the fourslash facilities? I was originally doing what you're asking, and was asked to move to doing this instead.

Reply:
I dont understand. Doesnt LS API take sourceFile, focus locations and text? 

I would think that it translates to verify(index.ts, focus ranges, text that you have right now in incoming.ts) is that not it?  I am saying the text of incoming.ts shouldnt be on the vfs in the test but passed to mapCode API as text itself. 

What i meant by different extension is to have the focus locations in ".js" for example